sssom/io.py

```diff
@@ -107,12 +114,11 @@ def validate_file(input_path: str) -> bool:
     :param input_path: The path to the input file in one of the legal formats, eg obographs, aligmentapi-xml
     :returns: True if valid SSSOM, false otherwise.
     """
-    try:
-        parse_sssom_table(file_path=input_path)
-        return True
-    except Exception as e:
-        logging.exception("The file is invalid", e)
-        return False
```

Comment:
remove the --> Bool  return type from validate_file(). Only `validate()` should have a return type boolean, validate_file just calls `validate()` as you already do here..